config/application.rb

```diff
@@ -19,11 +22,18 @@
 module OralHistory
   class Application < Rails::Application
     # Initialize configuration defaults for originally generated Rails version.
-    config.load_defaults 5.1
+    config.load_defaults 6.1
+    config.autoloader = :classic
```

Comment:
does classic mean spring here? I think that's fine, but we might sneak in zeitwork at some point